### DISCUSSION TITLE
Cannot find module '@babel/helper-validator-identifier'

### DISCUSSION THREAD

**Discussion by ismott** [babel/babel#15029] [Show and tell]
When trying to start a project in react-native I come across the following error

![image](https://user-images.githubusercontent.com/82410668/194651501-9b4ac208-e1c2-480a-be66-555da5fd13ce.png)

**Comment by nicolo-ribaudo** [comment]
Can you share the contents of the `package.json` of the `@babel/highlight` shown in that screenshot?
